mcs/class/corlib/System/Console.iOS.cs

```diff
@@ -7,7 +7,7 @@
 // Copyright 2012-2014 Xamarin Inc. All rights reserved.
 //
 
-#if FULL_AOT_RUNTIME
```

Comment:
Yeah technically this is built if you use the `mobile_static` profile, but it's not used because it only gets used in Console.cs `#if MONOTOUCH`.